### PR TITLE
Milestone 3: Portfolio Mobile Version

### PR DESCRIPTION
**Pull request by shahedakhatoon**
## Made the following changes:
1. Added a Contact Form in the Footer
2. Styled the background images
3. Styled the contents of the page

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -334,3 +334,77 @@ button {
...
+}
+
+.form-title {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  color: #fff;
+  text-align: center;
+}
```
Comment (keroloussamy) [inline style.css]: - [x] Kindly add some `padding-top`, to be like the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: style.css [hunk 2]
```diff
@@ -334,3 +334,77 @@ button {
...
+
+.content {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  color: #ebebff;
+  text-align: center;
+  margin: 10px;
+}
```
Comment (keroloussamy) [inline style.css]: - [x] Kindly make the `margin: 10px;` bigger, to be like the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: style.css [hunk 3]
```diff
@@ -334,3 +334,77 @@ button {
...
+  font-style: normal;
+  background-color: #fff;
+  font-weight: 500;
+  font-size: 17px;
+  border: 2px solid #ccc;
+  border-radius: 8px;
+  box-sizing: border-box;
+  width: 95%;
+  height: 150px;
+}
```
Comment (keroloussamy) [inline style.css]: - [x] Kindly you can make the `width: 95%;` smaller like just `90%` to be like the Figma design.
- [x] And maybe you can add some `margin-top` and `margin-bottom` to be like the Figma design.

image

File: style.css [hunk 4]
```diff
@@ -334,3 +334,77 @@ button {
...
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  border: 1px solid #ccc;
+  border-radius: 8px;
+  box-sizing: border-box;
+  padding: 12px;
+  margin-top: 10px;
+  margin-bottom: 8px;
+}
```
Comment (keroloussamy) [inline style.css]: - [x] Kindly add `width: 90%;` here to be like the Figma design. And like the message input.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @shahedakhatoon, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by prolajumokeoni** [comment]
## OPTIONAL
- [ ] Kindly fix the horizontal scroll in your website, you should avoid horizontal scrolls on your website, check your widths and margins 👍🏿 

https://user-images.githubusercontent.com/69638013/155287402-e938eddf-55b9-4457-a1d5-39527bf9ae4e.mp4

**Review by prolajumokeoni — APPROVED** [review]
Hi @shahedakhatoon 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
**_check optional comments_**

approved-stamp


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**